### ISSUE TITLE
In Firefox, when using the keyboard to play the piano keys, the 10th note is not playing.

### ISSUE THREAD

**Issue by sepidehshahi** [dinukadesilva/music-ctrls#89]
In Firefox, when using the keyboard to play the piano keys, the 10th note is not playing. Playing the 10th note also causes problems with moving the highlighted area with the arrow keys. 

To reproduce the issue:
1-Open the piano in Firefox
2-Bring the focus to the piano
3-Use keyboard buttons to play the piano white keys
4-when you reach the 10th note, it is not playing
5-after pressing the keyboard key for the 10th note, try using the arrow keys to move the highlighted area.

**Comment by dinukadesilva** [comment]
Hi @sepidehshahi 

Could you please verify again when the piano is disabled whether it's after 10th key or 11th key

For me, 

1. **10th key is not working at all in Firefox, But, that doesn't disable the keyboard.** 
The reason I found seems the key code is different in firefox
Firefox - 59
Chrome - 186

2. **11th key is working only once and then disables the keyboard as well.**
The reason is Firefox shortcut to open "Quick Find (links only)" is double quotes (keycode 222).
And then the focus is moved to that text field. As a result, the piano is not working until it's focused again.

![image](https://cloud.githubusercontent.com/assets/10680296/17436764/1c288a8e-5b37-11e6-8c0b-2f2435b1a47d.png)

**Comment by sepidehshahi** [comment]
That's right Dinuka. I tried it again. The 10th key is not working at all in the Firefox, however, if the 11th key is not played you can still use the keyboard after playing the 10th key. 
Thus, it's the 11th key that disables the keyboard. and as you have found out that's due to opening of the "Quick Find".

**Comment by dinukadesilva** [comment]
Thanks @sepidehshahi

**Comment by dinukadesilva** [comment]
@sepidehshahi @simonbates

There are two solutions for 10th key,

1. using `evt.key` property instead of `evt.keyCode.` But, there are some compatibility issues in that as described here https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key

2. Adding a conditional statement in the code to handle that. But, in that case, even the test script has to be updated with a conditional statement.

**Comment by dinukadesilva** [comment]
the bug related to the 11th key has been moved to another issue #95
